transforms/images/polus-imagej-filter-dog-plugin/plugin.json

```diff
@@ -0,0 +1,112 @@
+{
+  "name": "ImageJ filter dog",
+  "version": "0.3.0",
+  "title": "ImageJ filter dog",
+  "description": "This plugin applies the Difference of Gaussians algorithm to an input collection.",
+  "author": "Benjamin Houghton (benjamin.houghton@axleinfo.com)",
+  "institution": "National Center for Advancing Translational Sciences, National Institutes of Health",
+  "repository": "https://github.com/labshare/polus-plugins",
+  "website": "https://ncats.nih.gov/preclinical/core/informatics",
+  "citation": "",
+  "containerId": "polusai/imagej-filter-dog-plugin:0.3.0",
+  "inputs": [
+    {
+      "name": "opName",
+      "type": "enum",
+      "options": {
+        "values": ["DoGSingleSigmas",
+                    "DoGVaryingSigmas"]
+      },
+      "description": "Operation to peform",
+      "required": false
+    },
+    {
+      "name": "inpDIr",
+      "type": "collection",
+      "description": "The collection to be processed by this plugin",
+      "required": false
```

Comment:
Once again, it seems that these two arguments are always required.

Reply:
I have updated these to be required